plugins/techdocs-node/src/stages/publish/awsS3.test.ts

```diff
@@ -478,7 +465,31 @@ describe('AwsS3Publish', () => {
 
       await expect(fails).rejects.toMatchObject({
         message: expect.stringMatching(
-          'TechDocs metadata fetch failed; caused by Error: Unable to read stream; caused by Error: The file invalid/triplet/path/techdocs_metadata.json does not exist',
+          'TechDocs metadata fetch failed; caused by Error: The file invalid/triplet/path/techdocs_metadata.json does not exist',
```

Comment:
Maybe we can update the test to expect a string containing `The file invalid/triplet/path/techdocs_metadata.json does not exist`? Seems like that is the important part.

Reply:
Done